src/mcp/antigravity/README.md

```diff
@@ -0,0 +1,113 @@
+# Firebase MCP Server
+
+[![License](https://img.shields.io/badge/License-MIT-blue.svg)](../../../LICENSE)
+
+The Firebase Model Context Protocol (MCP) Server gives AI-powered development tools the ability to work with your Firebase projects and your app's codebase. The Firebase MCP server works with any tool that can act as an MCP client, including: Antigravity, [Firebase Studio](https://firebase.google.com/studio), [Gemini CLI](https://github.com/google-gemini/gemini-cli), [Claude Code](https://www.claude.com/product/claude-code), [Cline](https://github.com/cline/cline), [Cursor](https://www.cursor.com/), VS Code Copilot, [Windsurf](https://codeium.com/windsurf), and more!
```

Comment:
No need for this sentence - we are _only_ going to show this page on Antigravity.